testapi.pm

```diff
@@ -1025,11 +1031,18 @@ Special characters naming:
 =cut
 
 sub send_key {
-    my ($key, $do_wait) = @_;
-    $do_wait //= 0;
-    bmwqemu::log_call(key => $key, do_wait => $do_wait);
-    query_isotovideo('backend_send_key', {key => $key});
-    wait_idle() if $do_wait;
+    my $key = shift;
+    my %args = (@_ == 1) ? (do_wait => +shift()) : @_;
+    $args{do_wait}            //= 0;
+    $args{wait_screen_change} //= 0;
+    bmwqemu::log_call(key => $key, %args);
+    if ($args{wait_screen_change}) {
+        wait_screen_change { query_isotovideo('backend_send_key', {key => $key}) };
+    }
+    else {
+        query_isotovideo('backend_send_key', {key => $key});
+    }
+    wait_idle() if $args{do_wait};
```

Comment:
If we want to slowly remove wait_idle, why not also cluck if do_wait is > 0?.

Reply:
because that is already within wait_idle, isn't it?